puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/triggers/github_five_hours.yaml

```diff
@@ -0,0 +1,6 @@
+- trigger:
+    name: github_five_hours
+    triggers:
+      - github
+      - pollscm: '*/5 * * * *'
+      - schedule-failed-builds: '* * * * *'
```

Comment:
This might not be useful if this scm config is to be reused, we only use it on a few jobs - generally we don't retry.